### PR TITLE
Support ubuntu 22.04 in deployment of testbed

### PR DESCRIPTION
**Pull request by ZhaohuiS**
Signed-off-by: Zhaohui Sun <zhaohuisun@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Support ubuntu 22.04 in deployment of testbed.

### CODE REVIEW

File: ansible/roles/vm_set/tasks/main.yml [hunk 1]
```diff
@@ -98,18 +98,18 @@
...
   set_fact:
     pip_executable: pip
-  when: pip_executable is not defined and host_distribution_version.stdout != "20.04"
+  when: pip_executable is not defined and host_distribution_version.stdout != "20.04" and host_distribution_version.stdout != "22.04"
 
 - name: Get default pip_executable
   set_fact:
     pip_executable: pip3
-  when: pip_executable is not defined and host_distribution_version.stdout == "20.04"
+  when: pip_executable is not defined and host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04"
```
Comment (wangxin) [inline ansible/roles/vm_set/tasks/main.yml:112]: Should the condition be like this:
```
when: pip_executable is not defined and (host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04")
```

Comment (ZhaohuiS) [reply ansible/roles/vm_set/tasks/main.yml]: > Should the condition be like this:
> 
> ```
> when: pip_executable is not defined and (host_distribution_version.stdout == "20.04" or host_distribution_version.stdout == "22.04")
> ```

@wangxin Thanks for pointing it out. Done.

**Review by wangxin — APPROVED** [review]

**Pull request merged**